crates/sp-domains/src/verification.rs

```diff
@@ -31,10 +31,11 @@ pub enum VerificationError {
     InvalidBundleDigest,
 }
 
+/// Type that provides utilities to verify the storage proof.
 pub struct StorageProofVerifier<H: Hasher>(PhantomData<H>);
 
 impl<H: Hasher> StorageProofVerifier<H> {
-    pub fn verify_and_get_value<V: Decode>(
+    pub fn get_decoded_value<V: Decode>(
```

Comment:
The first part of this func is duplicated with `get_bare_value`, perhaps using `get_bare_value` instead.